docs: Add example Envoy escape hatch configs (#12764)
hashicorp/consul: website/content/docs/connect/proxies/envoy.mdx

  that will be appended to the array of [static
  clusters](https://www.envoyproxy.io/docs/envoy/v1.17.2/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-field-config-bootstrap-v3-bootstrap-staticresources-clusters)

>   in the bootstrap config. This allows adding custom clusters for tracing sinks

Reviewer: ```suggestion
  in the bootstrap config. This enables you to add custom clusters for tracing sinks,
```

>   for example. For a single cluster just, encode a single object. For multiple

Reviewer: ```suggestion
  for example. Encode a single object for a single cluster. For multiple
```
I'm a little unclear, is this what we mean?

Author: ```suggestion
  for example. In order to configure a single cluster, specify a single JSON object with the cluster details. For multiple
```

Author: This is closer to what this sentence was trying to convey.

>   clusters, the objects should be comma separated with no trailing comma. The

Reviewer: ```suggestion
  clusters, specify objects in a comma-separated list with no trailing comma. The
```
